src/tunacode/ui/renderers/tools/base.py

```
@@ -76,7 +76,32 @@ def truncate_content(
     return "\n".join(truncated), max_lines, total
 
 
-def pad_lines(lines: list[str], min_lines: int = MIN_VIEWPORT_LINES) -> list[str]:
+def parse_match_line(match_line: Any) -> dict[str, Any]:
```

Ruff (0.14.14)

Dynamically typed expressions (typing.Any) are disallowed in `match_line`

(ANN401)